Source/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -12,16 +12,14 @@ import XCTest
 
 class ConfigurationTests: XCTestCase {
     func testInit() {
-        XCTAssert(Configuration(yaml: "") != nil,
-            "initializing Configuration with empty YAML string should succeed")
-        XCTAssert(Configuration(yaml: "a: 1\nb: 2") != nil,
-            "initializing Configuration with valid YAML string should succeed")
-        XCTAssert(Configuration(yaml: "|\na") == nil,
-            "initializing Configuration with invalid YAML string should fail")
+        XCTAssert(Configuration(dict: [:]) != nil,
+            "initializing Configuration with empty Dictionary should succeed")
+        XCTAssert(Configuration(dict: ["a" : 1, "b" : 2]) != nil,
```

Comment:
Please use no space to the left, and a single space to the right, of colons when using dictionary literals to stay consistent with the style of the rest of the project.